### PR TITLE
docs: Install with conan doc pages

### PR DESCRIPTION
**Pull request by MikeRavenelle**
Documentation for conan support.

- Added conan install documentation and modified section to include it.

## Documentation change checklist

- [ X] My documentation changes follow the [docs style guide](https://dpp.dev/docs-standards.html) and any code examples follow the [coding style guide](https://dpp.dev/coding-standards.html).
- [X] I tested that my change works before raising the PR (via running `doxygen`, and testing examples).
- [X] I have not moved any existing pages or changed any existing URLs without strong justification as to why.
- [X] I have not generated content using AI or a desktop utility such as grammarly.

### CODE REVIEW

**Comment by braindigitalis** [comment]
reopened and targeted at dev

**Review by braindigitalis — APPROVED** [review]
good to go as soon as Conan is live 👍

**Comment by Jaskowicz1** [comment]
Renamed PR to correctly align with our conventions

**Comment by Jaskowicz1** [comment]
Renamed this PR to correctly align with our conventions.

File: docpages/install/install-conan.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+```
+set(CMAKE_BUILD_TYPE Release)
+```
+
+- Now run the following commands
+
+```
+mkdir build && cd build
+conan install .. --build=missing -of=. -s build_type=Release
```
Comment (Jaskowicz1) [inline docpages/install/install-conan.md:47]: This doesn't work on Windows. It seems you can't specify the path if you have `-of=.`.

Doing `conan install .. --build=missing -s build_type=Debug` seems to fix this.

**Review by Jaskowicz1 — CHANGES_REQUESTED** [review]

File: docpages/install/install-conan.md (hunk 1 above)
Comment (Jaskowicz1) [reply docpages/install/install-conan.md:47]: Hm, you do need the `-of=` so I would maybe change that to just be the current path.

Comment (Jaskowicz1) [reply docpages/install/install-conan.md:47]: Scrap that, I would change this entire thing to be something like the following:
```bash
mkdir build
conan install . --output-folder=build --build=missing -s build_type=Release
cd build
cmake .. -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake"
cmake --build build -j8 --config Release
```

**Pull request merged**